### ISSUE TITLE
Cannot generate default format documentation

### ISSUE THREAD

**Issue by MarcSteven** [SwiftDocOrg/DocTest#5]
Thank for your sharing @mattt , but I just tried and then cannot generate the default format documentation

**Comment by mattt** [comment]
Sorry, but I'm having trouble understanding what you mean by this. Could you please be more specific about what you did, what you expected, and what actually happened?

**Comment by MarcSteven** [comment]
Yeah it cannot generate the default format when I execute the command
`swift doc generate /Users/MarcZhao/path  --module-name ExampleTest`

**Comment by mattt** [comment]
Transferring this to @SwiftDocOrg/swift-doc, as this doesn't seem to be related to DocTest.
